authors/apps/authentication/tests/test_signup.py

```diff
@@ -89,3 +81,27 @@ def test_registeration_for_a_super_user(self):
         self.assertEqual(admin_user.is_active, True)
         self.assertEqual(admin_user.is_staff, True)
         self.assertEqual(admin_user.is_superuser, True)
+
```

Comment:
Test for all possible errors.

Reply:
No need for those, they'll be tested when we test for permissions for accessing endpoints.